src/main/scripts/update_versions.py

```diff
@@ -0,0 +1,75 @@
+import logging
+
+import requests
```

Comment:
This is just a script to help with the update for the next person doing this. It still needs a bit of handholding (checking the LTS, installing these 2 prerequisites) but I still think it makes it much easier.

Reply:
We can fetch the LTS product version automatically. There is a python script we have (`./src/test/scripts/product_versions.py`) and we can get use like this:
```
import product_version
...
product = "bamboo"
...
lts_version = product_versions.get_lts_version(product)
...
```